insolar/jet/fetcher.go

```diff
@@ -301,9 +301,17 @@ func (tu *fetcher) nodesForPulse(ctx context.Context, pulse insolar.PulseNumber)
 
 	res, err := tu.Nodes.InRole(pulse, insolar.StaticRoleLightMaterial)
```

Comment:
This will return `node.ErrNoNodes`. We need to catch it and **only then** return heavy node for **current pulse**.

Reply:
Also, please write a test for this case.

Reply:
I think this logic is not correct. When we have just started on existing db, we will have Light nodes, but all of them might not have actual jets.
About tests - yes, sure!